### PR TITLE
Add historical data for sheets that didn't have historical data in 2015

### PR DESCRIPTION
**Pull request by esirK**
## Description

Adds historical data for sheets that didn't have historical data in 2015

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
Screenshot 2021-02-24 at 13 13 59

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by esirK** [comment]
@kilemensi  I'm making this a draft for now because I will be adding all sheets that didn't have any historical data in 2015 here.

**Comment by esirK** [comment]
@kilemensi I think we can review this as the first batch as I work on the next batch. In sheet 32 for example, I need to extract the 2015 data from the report since it's not in the `historical.json` file.

File: reports/report_builder.py [hunk 1]
```diff
@@ -4600,7 +4613,13 @@ def tabulate_historical(self, ws, current_ws, cols, rows, c=None, r=6, write_row
...
 
                         if value is None:
-                            value = ['n/a'] * values_per_col
+                            # check if it's due to having % and N
+                            p = data.get(canon(col_heading), {}).get('%', {})
+                            n = data.get(canon(col_heading), {}).get('n', {})
+                            if (p or n):
+                                value = [p.get(canon(row_heading), "n/a"), n.get(canon(row_heading), "n/a")]
+                            else:
+                                value = ['n/a'] * values_per_col
```
Comment (kilemensi) [inline reports/report_builder.py:4622]: Can you explain a bit how this works / why wasn't it done before?

**Review by kilemensi — APPROVED** [review]
🚀

File: reports/report_builder.py (hunk 1 above)
Comment (esirK) [reply reports/report_builder.py:4622]: From what I've seen, in 2015, we didn't have any historical data that contained 2 values i.e `%` and `n`. Most of them(2010, 2005 data) were just in the following format
```
{'africa': 0.44, 'asia': 0.49, 'caribbean': 0.34, 'europe': 0.64, 'latin america': 0.44, 'middle east': 0.34, 'north america': 0.83, 'pacific islands': 0.43}
```
In 2015, data containing `%` and `n` was added and that is what we now need to extract.
Basically, this part of the code helps extract values that are one level down 
`{'internet': {'%': {'celebrity, arts and media, sports': 0.182608695652174, ...}, 'n': {'celebrity, arts and media, sports': 84, ...}}, `
@kilemensi

**Pull request merged**